:memo: Clarify generic CDN setup (#2867)
platformsh/platformsh-docs: docs/src/domains/steps/_index.md

1. Open your CDN's management system.
2. Point the CDN at your [target](#2-get-the-target-for-your-project).
3. Open your registrar’s domain management system.

> 4. Configure your DNS zone settings to point at your CDN.

Reviewer: 🛠️  I think steps 3 and 4 can be joined together to shorten the procedure.

```suggestion
3. Open your registrar’s domain management system and configure your DNS zone settings to point at your CDN.
```

> 3. Open your registrar’s domain management system and configure your DNS zone settings to point at your CDN.

Reviewer 2: :clipboard: 
```suggestion
4. Open your registrar's domain management system and configure your DNS zone settings to point at your CDN.
```

   The address or `CNAME` record to use varies by CDN provider.
   Refer to the official documentation of your DNS provider and CDN provider.

> 5. Check that redirects and subdomains are set correctly for the [TLS certificate ownership verification](../troubleshoot.md#ownership-verification).

Reviewer: 🛠️ To keep in line with previous suggestion (numbering). 

```suggestion
4. Check that redirects and subdomains are set correctly for the [TLS certificate ownership verification](../troubleshoot.md#ownership-verification).
```

> 4. Check that redirects and subdomains are set correctly for the [TLS certificate ownership verification](../troubleshoot.md#ownership-verification).

Reviewer 2: :clipboard: 
```suggestion
5. Check that redirects and subdomains are set correctly for the [TLS certificate ownership verification](../troubleshoot.md#ownership-verification).
```

> 6. [Disable the router cache](../cdn/_index.md#disable-the-platformsh-router-cache).

Reviewer: 🛠️ To keep in line with previous suggestion (numbering). 

```suggestion
5. [Disable the router cache](../cdn/_index.md#disable-the-platformsh-router-cache).
```

> 7. Optional: For increased security and avoid the CDN to be bypassed,

Reviewer: 🛠️  Much better with added context 👍 

```suggestion
6. Optional: For increased security and prevent the CDN from being bypassed,
```

   you can force all traffic to [go through the CDN](../cdn/_index.md#prevent-direct-access-to-your-platformsh-server).

> 8. Optional: If you have multiple domains you want to be served by the same app, add a `CNAME` record for each of them.

Reviewer: 🛠️ To keep in line with previous suggestion (numbering). 

```suggestion
7. Optional: If you have multiple domains you want to be served by the same app, add a `CNAME` record for each of them.
```

> 5. [Disable the router cache](../cdn/_index.md#disable-the-platformsh-router-cache).
> 6. Optional: For increased security and to prevent the CDN from being bypassed,
>    you can force all traffic to [go through the CDN](../cdn/_index.md#prevent-direct-access-to-your-platformsh-server).
> 7. Optional: If you have multiple domains you want to be served by the same app, add a `CNAME` record for each of them.

Reviewer 2: ```suggestion
6. [Disable the router cache](../cdn/_index.md#disable-the-platformsh-router-cache).
7. Optional: For increased security and to prevent the CDN from being bypassed,
   you can force all traffic to [go through the CDN](../cdn/_index.md#prevent-direct-access-to-your-platformsh-server).
8. Optional: If you have multiple domains you want to be served by the same app, add a `CNAME` record for each of them.
```

   That includes the `www` subdomain if you are using it in your [routes configuration](../../define-routes/_index.md).

> Using a CDN with a custom domain prevents [the `X-Robots-Tag` from being added to requests](../../environments/search-engine-visibility.md#how-its-done).

Reviewer 2: :hammer_and_wrench: 
```suggestion
Adding a custom domain sets your site as [visible to search engines](../../environments/search-engine-visibility.md#how-its-done).
```

> For more in depth-configuration options, check the [CDN guide](../cdn/_index.md).

Reviewer: 🛠️ To better stick with our style guide.

```suggestion
For more information, see how you can further [configure your CDN] (../cdn/_index.md).
```

> For more information, see how you can further [configure your CDN](../cdn/_index.md).

Reviewer 2: :hammer_and_wrench: 
```suggestion
See how you can further [configure your CDN](../cdn/_index.md).
```
